frontend/src/variables.scss

```diff
@@ -75,4 +75,12 @@ $shadow: 0 0.5em 1em -0.125em rgba($scheme-main, 0.1), 0 0px 0 1px rgba($scheme-
 
 $scheme-invert: $white;
 $scheme-invert-bis: $white-bis;
-$scheme-invert-ter: $white-ter;
\ No newline at end of file
+$scheme-invert-ter: $white-ter;
+
+
+$radius-small: 0.3em;
+
+$modal-background-background-color: rgba($scheme-main, 0.86);
+
+$input-background-color: #D9D9D9;
+$input-color: $black;
```

Comment:
I'd use `$input-color: $dark;` just for the sake of consistency - although the two colors are not really that different...